src/lib.rs

```diff
@@ -107,11 +109,12 @@ impl TargetDep {
                 .expect("output missing trailing :");
 
             // Move output
-            std::fs::rename(out_path, &self.output_path).unwrap_or_else(|err| {
-                panic!(
-                    "Failed to move output {:?} to {:?}: {}",
-                    out_path, &self.output_path, err
-                )
+            std::fs::copy(out_path, &self.output_path).unwrap_or_else(|err| {
+                panic!("error when copying file: {:?}", err)
+            });
+
+            std::fs::remove_file(out_path).unwrap_or_else(|err| {
+                panic!("error when removing file: {:?}", err)
```

Comment:
Probably not important to remove. The rename here is just for performance, not because it needs "move" semantics.

Reply:
removed the `remove_file` logic